pkg/agent/manager/private.go

```diff
@@ -0,0 +1,115 @@
+package manager
```

Comment:
I found it a little confusing that these methods were somewhere other than `manager.go`

Reply:
I think you are right...I thought at first that it would be "cool" to have all the unexported method in a separate file....but then when I started navigating through the code I felt the same you are commenting. I will move them back to manager.go.